.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.1.14
+  hooks:
+    # Run the linter.
+  - id: ruff
+    # Run the formatter.
+  - id: ruff-format
+    args: [--check]
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-yaml
+  - id: check-toml
+  - id: check-ast
+  - id: check-added-large-files
+  - id: check-merge-conflict
```

Comment:
How does `check-merge-conflict` prevent merge conflicts? I don't see how it would work.

Reply:
Check for files that contain [merge conflict strings](https://github.com/pre-commit/pre-commit-hooks/blob/77c7147e528b9a7c20f3546a6de3ac0cf6f15859/pre_commit_hooks/check_merge_conflict.py#L10).

Reply:
Ok, I see. Then it is fine.